zulip/integrations/matrix/test_matrix.py

```diff
@@ -78,40 +78,37 @@ def test_write_sample_config(self):
 
 
 class MatrixBridgeZulipToMatrixTests(TestCase):
+    valid_zulip_config = dict(
+        stream="some stream",
+        topic="some topic",
+        email="some@email"
+    )
+    valid_msg = dict(
+        sender_email="John@Smith.smith",  # must not be equal to config:email
+        type="stream",  # Can only mirror Zulip streams
+        display_recipient=valid_zulip_config['stream'],
+        subject=valid_zulip_config['topic']
+    )
```

Comment:
I feel that the previous commit and this commit should be squashed as this change is pretty minor.

Reply:
The previous commit adds tests; this commit is a refactor. I would be happy to discuss, but I feel that these are separate.